### ISSUE TITLE
api.webtor.buzz blocked by Cloudflare Malware Blocking DNS (1.1.1.2 and 1.1.1.3)

### ISSUE THREAD

**Issue by quinncomendant** [webtor-io/web-ui-v1#5]
webtor.io is not working. I'm getting `net::ERR_NAME_NOT_RESOLVED` errors in the console logs. It seems the problem is because api.webtor.buzz is blocked by my DNS host, [1.1.1.3](https://blog.cloudflare.com/introducing-1-1-1-1-for-families/):

```
❯ host api.webtor.buzz
api.webtor.buzz has address 0.0.0.0
api.webtor.buzz has IPv6 address ::
```

I have reported this false positive to Cloudflare's NetOps team (done here: https://report.teams.cloudflare.com/) as categories `P2P` and `File Sharing`:

![Screen Shot 2022-05-25 at 16 39 50](https://user-images.githubusercontent.com/6884276/170371915-a8ddd5be-d438-458c-8eef-03d3aac03537.png)

Since this is not guaranteed to get a response, you might consider using an alternate domain or IP addresses directly.

**Comment by vintikzzz** [comment]
Submit is blocked for me...
![image](https://user-images.githubusercontent.com/150981/170381374-1290b9c5-554e-430a-b5da-c01d0ffe4d00.png)
It is strange, they haven't even send me any abuses or so...

**Comment by vintikzzz** [comment]
Ok, I've done as you suggested.

**Comment by metal1one** [comment]
turned off defender and antivirus and still getting the error. Anything else needed to be done to get website working again?

**Comment by vintikzzz** [comment]
@metal1one may I ask you to check network panel in Google Chrome developer tools and check network panel?
I have attached screenshot of successful result.
 
![image](https://user-images.githubusercontent.com/150981/171579308-9005f9fd-13b6-4082-a716-d1df102c1028.png)

**Comment by metal1one** [comment]
![image](https://user-images.githubusercontent.com/17607429/171631608-85c81b25-f092-4470-bad2-795cd38f089c.png)

**Comment by vintikzzz** [comment]
@metal1one just visit https://api.webtor.buzz/ directly and make a screen of an error. And also copy your DNS servers ips...

**Comment by metal1one** [comment]
> @metal1one just visit https://api.webtor.buzz/ directly and make a screen of an error. And also copy your DNS servers ips...

Yeah, it appears it is being blocked by whatever DNS my ISP is giving me... changing to googles dns servers and it works.

**Comment by quinncomendant** [comment]
Yeah, Cloudflare malware-blocking DNS (1.1.1.2) is still blocking api.webtor.buzz. You can test this on the command line by running `dig @1.1.1.2 api.webtor.buzz +short` and if the output is `0.0.0.0` it is blocked:

```sh
❯ dig @1.1.1.2 api.webtor.buzz +short
0.0.0.0
```

If you're on linux or mac, you can just add the correct IP address to your `/etc/hosts` file, which supersedes DNS:
```
138.201.33.57 api.webtor.buzz
```

You can help by submitting a request to reclassify api.webtor.buzz with the Cloudflare NetOps team:

1. Open https://report.teams.cloudflare.com/
2. Select `P2P` and `Fire Sharing`
3. Deselect `Malware`
4. Submit

**Comment by vintikzzz** [comment]
I am thinking about rotating domains for API (they are quite cheap nowadays). But I don't know how fast my domains would get blocked... I am afraid if these blocks are performed automatic, I would not be able to rotate it with appropriate speed...
